api/src/main/scala/Utils.scala

```diff
@@ -203,10 +203,12 @@ trait Utils {
   def joinCollectionLayers(
     layers: Seq[TileLayerCollection[SpatialKey]]
   ): Map[SpatialKey, Seq[Tile]] = {
+    val layout = layers.head.metadata.layout
+    val emptyTile = IntConstantTile(NODATA, layout.tileCols, layout.tileRows)
```

Comment:
Just to share a thought (no action required) - One thing that I considered was recommending this be a `lazy val` - then it would not be constructed if there were not cases of empty tiles. However, the savings there would be very scant as a constant tile has a small footprint, so that would be over-optimizing.

Reply:
Ah great tip. I'd like to get that in, because even if the savings are scant, it sounds like good practice within Scala.

Reply:
Fixed in 35c9252.